solr/packaging/test/test_adminconsole_urls.bats

```diff
@@ -25,7 +25,7 @@ teardown() {
   # save a snapshot of SOLR_HOME for failed tests
   save_home_on_failure
 
-  solr stop -all >/dev/null 2>&1
+  solr stop --all >/dev/null 2>&1
```

Comment:
I honestly stayed away from touching `bin/solr start` and `bin/solr stop` since I suspect lots of folks have scripts that call those and otherwise it seemed complicated.   I am up for changing `-all` to `--all`, however, can we augment the bats tests to try both?  Just to verify that `-all` and `-all` both work?

Reply:
I think adding tests in addition that checks that both still work is a good idea.

As far as I understand, the function "teardown" is responsible for test cleanup, so I guess it can be updated without problems? Or is there a chance that the test will be executed with a version that does not support `--all`?